src/store/createStore.js

```diff
@@ -1,15 +1,15 @@
 import { applyMiddleware, compose, createStore } from 'redux'
 import { routerMiddleware } from 'react-router-redux'
-import thunk from 'redux-thunk'
+import {reduxObservable} from 'redux-observable'
```

Comment:
I have seen createEpicMiddleware and looking at updating this example as thunkservables are not deprecated.